server/api/utils/singletons/k8s.py

```diff
@@ -540,6 +540,65 @@ def delete_secrets(
         self.v1api.delete_namespaced_secret(secret_name, namespace)
         return mlrun.common.schemas.SecretEventActions.deleted
 
+    @raise_for_status_code
+    def create_configmap(
```

Comment:
```suggestion
    def ensure_configmap(
```
or another name, as what you do here is create/replace and not just "create"